tools/NuGet/BuildDynamoCore.bat

```diff
@@ -0,0 +1,29 @@
+SET base=..\..\bin\AnyCPU\Debug
```

Comment:
A new NuGet has been added called `DynamoVisualProgramming.Core`. This is the build file for that NuGet.